app/assets/locales/locale-en.json

```diff
@@ -1708,8 +1708,48 @@
+                "text_legend": "Force settlement: Holders of a SmartCoin can request a force settlement. The SmartCoin is used to forcefully pay back the debt of the least collaterized positions while the former SmartCoin holder receives collateral as if he sold it for the force settlement price. Any leftover collateral is returned to the position holder. This mechanism has an enforced delay after being triggered to avoid abuse.\nForce Settlement Price: The price at which a force settlement will happens. It is calculated from the feed price by adding a force settlement offset to disincentivize force settlements."
+            },
+            "steps_setup": {
+                "title": "Setup",
+                "title_within": "How does it work?",
+                "text": "You decide how much collateral you want to provide and what the collateral ratio (CR) should be, only lower bound is the maintenance (minimum) collateral ratio (MCR). You spend the SmartCoin as you which, and when you no longer need the liquidity you buy it back and close the position by paying your debt.",
+                "text_legend": "Collateral Ratio (CR): The CR defines how much percent of the generated SmartCoin value is covered by collateral.\nMaintenance Collateral ratio (MCR): To guarantee that the SmartCoin has enough backing collateral and to account for volatility, the MCR is enforced when creating the position."
+            },
+            "steps_risks": {
+                "title": "Risks",
+                "title_within": "What are the risks involved in borrowing?",
+                "text": "As long as you monitor your position and make sure that the collateral ratio (CR) stays above the maintenance collateral ratio (MCR), your position will not be liquidated. If the value of the collateral drops and your position gets close to MCR, you can lock up more collateral or pay back some of the debt to increase the CR. Failure to remain above MCR results in force liquidation of your position (margin call). This means that your collateral is being sold automatically on the respective market to reduce your debt, with a penalty to ensure the order is filled (see MSSR). Any leftover collateral is returned to you after the full debt is paid. You can choose to only allow partial selling of your position by setting the target collateral ratio (TCR). If your position can not be force liquidated due to extreme market conditions and the CR drops below the maximum short squeeze ratio (MSSR), the SmartCoin automatically enters global settlement state and you lose your position instantly.",
```

Comment:
```suggestion
                "text": "As long as you monitor your position and make sure that the collateral ratio (CR) stays above the maintenance collateral ratio (MCR), your position will not be liquidated. If the value of the collateral drops and your position gets close to MCR, you can lock up more collateral or pay back some of the debt to increase the CR. Failure to remain above MCR results in a forced liquidation of your position (margin call). This means that your collateral is automatically sold on the respective market to reduce your debt, with a penalty, to ensure the order is filled because the Maximum Short Squeeze Ratio (MSSR), a debt position has been margin called. Any leftover collateral is converted to BTS and returned to you after the full debt is paid. You can choose to only allow partial selling of your position by setting the Target Collateral Ratio (TCR). If your position can not be forced to liquidate due to extreme market conditions and the Collateral Ratio (CR) drops below the maximum short squeeze ratio (MSSR), the SmartCoin (MPA) automatically enters a state of Global Settlement (GS) and you lose your entire position instantly.",
```